api/apps/api/test/projects/project-feature-tags/project-tags.e2e-spec.ts

```diff
@@ -123,9 +124,30 @@ describe('Projects Tag GET (e2e)', () => {
       projectId1,
       'name13',
     );
+    const featureIdRunning = await fixtures.GivenFeatureOnProject(
+      projectId1,
+      'running',
+      JobStatus.running,
+    );
+    const featureIdFailure = await fixtures.GivenFeatureOnProject(
+      projectId1,
+      'failure',
+      JobStatus.failure,
+    );
+
     await fixtures.GivenTagOnFeature(projectId1, featureId11, 'OneRepeatedTag');
     await fixtures.GivenTagOnFeature(projectId1, featureId12, 'OneRepeatedTag');
     await fixtures.GivenTagOnFeature(projectId1, featureId13, 'AnotherTag');
+    await fixtures.GivenTagOnFeature(
+      projectId1,
+      featureIdRunning,
+      'Irrelevant',
+    );
+    await fixtures.GivenTagOnFeature(
+      projectId1,
+      featureIdFailure,
+      'Irrelevant',
+    );
```

Comment:
the intention of these fixtures is quite clear - please consider whether you want to make the assertion part more explicit, by asserting that `response.body.data` does _not_ contain `Irrelevant` as tag.

Reply:
In line 161 there's this assertion:
`expect(response.body.data).toEqual(['AnotherTag', 'OneRepeatedTag']);`
It should cover that case, right?